src/pages/Register.vue

```diff
@@ -91,10 +91,7 @@ body
         &.spacer
           margin-bottom: 15px
       button
-        background-color: $secondary
-        color: WHITE
-        height: 30px
-        line-height: 30px
+        height: 40px
```

Comment:
We should consider moving buttons into a global file. It looks like we redefine the button for each component.

Reply:
Yes, that's part of #17.